rs/nns/integration_tests/BUILD.bazel

```diff
@@ -207,7 +208,7 @@ rust_canister(
     aliases = ALIASES,
     proc_macro_deps = MACRO_DEPENDENCIES,
     service_file = ":test_canisters/empty.did",
-    deps = DEPENDENCIES,
+    deps = DEPENDENCIES + ["//rs/nns/gtc"],
```

Comment:
We (i.e. the whole ic repo) generally do

```
deps = DEPENDENCIES,
```

no, ` + ["whatever"]`

I do not see why we are suddenly not doing that here.